compatibility-verifier/sample-test-suite/pre-controller-upgrade.yaml

```diff
@@ -33,7 +39,23 @@ operations:
     tableConfigFileName: feature-test-1.json
     recordReaderConfigFileName: data/recordReaderConfig.json
     segmentName: FeatureTest1_Segment
+  - type: streamOp
+    description: create Kafka topic PinotRealtimeFeatureTest2Event
+    op: CREATE
+    streamConfigFileName: feature-test-2-realtime-stream-config.json
```

Comment:
Make sure we have 1 partition in the stream.

Reply:
feature-test-2-realtime-stream-config.json has 1 partition in the stream.